libs/designer/src/lib/ui/templates/cards/templateCard.tsx

```diff
@@ -162,6 +169,52 @@ export const TemplateCard = ({ templateName }: TemplateCardProps) => {
   );
 };
 
+export const BlankWorkflowTemplateCard = () => {
+  const intl = useIntl();
+
+  const intlText = {
+    TEMPLATE_LOADING: intl.formatMessage({ defaultMessage: 'Loading....', description: 'Loading text', id: 'cZ60Tk' }),
+    NO_CONNECTORS: intl.formatMessage({
+      defaultMessage: 'This template does not have connectors',
+      description: 'Accessibility text to inform user this template does not contain connectors',
+      id: 'aI9W5L',
+    }),
+    BLANK_WORKFLOW: intl.formatMessage({
+      defaultMessage: 'Blank workflow',
+      description: 'Title text for the card that lets users start from a blank workflow',
+      id: 'pykp8c',
+    }),
+    BLANK_WORKFLOW_DESCRIPTION: intl.formatMessage({
+      defaultMessage: 'Start with a blank workflow to build your integration process from scratch.',
+      description: 'Label text for the card that lets users start from a blank workflow',
+      id: 'nN1ezT',
+    }),
+  };
+
+  const onBlankWorkflowClick = () => {
+    TemplateService().onAddBlankWorkflow();
```

Comment:
Lets add some telemetry whenever this is getting clicked.

Reply:
Added a telemetry aligning with the other templateCard